libyul/AsmParser.cpp

```diff
@@ -131,21 +177,21 @@ Statement Parser::parseStatement()
 		return parseForLoop();
 	case Token::Break:
 	{
-		Statement stmt{createWithLocation<Break>()};
+		Break stmt = createWithLocation<Break>();
```

Comment:
Why the change?

Reply:
was left over from some old code that required it. reverted.